### DISCUSSION TITLE
How to import explicitly $fetch in nuxt 3

### DISCUSSION THREAD

**Discussion by SujithGunasekaran** [nuxt/nuxt#16462] [Questions] [answered]
Hi Team,

Currently, I am using nuxt 3 for one of my projects, I can explicitly import nuxt composable, vue methods from _#imports_ But I can't able to import _$fetch_. How can I able to explicitly import _$fetch_ in nuxt 3 ?

```
import { reactive, useHead } from '#imports';
```

Can someone please help me

**Comment by TheAlexLichter** [comment, marked as answer]
`import { $fetch } from 'ohmyfetch'` coming from https://github.com/unjs/ohmyfetch

**Reply by SujithGunasekaran** [reply]
Thanks @manniL
